doc/esmvalcore/config.rst

```diff
@@ -46,23 +180,29 @@ As an example, the CMIP5 file path on BADC would be:
 
         [institute]/[dataset ]/[exp]/[frequency]/[modeling_realm]/[mip]/[ensemble]/latest/[short_name]
 
-When loading these files, ESMValTool replaces the placeholders with the true
-values. The resulting real path would look something like this:
+When loading these files, ESMValTool replaces the placeholders ``[item]`` with
+actual values supplied for by the user in ``config-user.yml`` and
+``recipe.yml``. The resulting real path would look something like this:
 
-.. code-block:: yaml
+.. code-block::
 
     MOHC/HadGEM2-CC/rcp85/mon/ocean/Omon/r1i1p1/latest/tos
 
+Again, for a more in-depth description this process, as part of the data
```

Comment:
Is there an "of" missing? "description of this process"?

Reply:
```suggestion
Again, for a more in-depth description of this process, as part of the data
```